install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -289,7 +289,8 @@ startAppOnK8s() {
     # restart ui container to pick up new TLS certs from daemon
     local replPod=$(kubectl get pods --selector='app=replicated,tier=master' | tail -1 | awk '{ print $1 }')
     if [ -n "$replPod" ]; then
-        kubectl exec "$replPod" -c replicated-ui -- kill 1
+        kubectl exec "$replPod" -c replicated-ui -- kill 1 || true
+        sleep 30
```

Comment:
30 seconds is a long time.... wanna do something like `echo "Waiting for replicated-ui to terminate"`?

Reply:
ok